spec/index.html

```diff
@@ -261,7 +261,7 @@ <h2>Notation and Terminology</h2>
     <p>A <dfn>ground</dfn> RDF graph is one that contains no blank nodes.</p>
 
     <p>Suppose that M is a functional mapping from a set of blank
-      nodes to some set of literals, blank nodes and IRIs. Any graph obtained
+      nodes to some set of RDF terms. Any graph obtained
```

Comment:
```suggestion
      nodes to some set of [=RDF terms=]. Any graph obtained
```
This currently creates a Respec error, but should IMO be included. I created an issue in RDF-concepts so that the error disappears.